pkg/util/util.go

```diff
@@ -24,7 +24,7 @@ const (
 
 var (
 	maskPasswordRegex1 = regexp.MustCompile(`[p|P]assword=(\s*?)'([^']+?)'`)
```

Comment:
Shouldn't this be space consumable as well?

Reply:
This works because the class is `NOT '` which includes a space